### PR TITLE
adding first version of configuration file

### PR DESCRIPTION
**Pull request by MarZajac**
adding first version of configuration file

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: config.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# set key and value
+date=yyyy-MM-dd
```
Comment (rafalmisiak) [inline config.properties]: możemy zmienić nazwlę kluczana coś w stylu dateFormat? Bo samo date sugeruje że jest to data a nie jej format

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Pull request merged**